torch/ao/quantization/fx/prepare.py

```diff
@@ -1279,6 +1285,7 @@ def insert_observers_for_model(
 
                             is_observer_in_same_graph_ = is_observer_in_same_graph(node, modules, node_name_to_target_dtype)
 
+                        if maybe_output_obs_node is not None or _is_custom_module_lstm(node, modules):
```

Comment:
what does this check mean?

Reply:
Since we no longer return an output observer for LSTM in L1261, we can skip updating the graph to point to the new observer (L1264-L1284), but we still need to swap the custom module, so we do a check for LSTM here.

Reply:
But actually maybe this line should be right above L1297 instead. Let me move it